Bottom: 40559755ac3ae93f13572f97d69f3dba95d61ffb
Top:    b49d73f00dab626a6e441962235bd4c7204ba718
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-09-28 13:28:50 +0800

Refresh of e1000-use-alias-for-default

---

diff --git a/hw/pci/pci.c b/hw/pci/pci.c
index a6f3eb2..c36a37e 100644
--- a/hw/pci/pci.c
+++ b/hw/pci/pci.c
@@ -1637,7 +1637,6 @@ static const char * const pci_nic_models[] = {
     "i82557b",
     "i82559er",
     "rtl8139",
-    "e1000",
     "e1000-82540em",
     "pcnet",
     "virtio",
